app/views/nonprofits/cards/edit.html.erb

```diff
@@ -22,7 +22,6 @@
       recaptcha_site_key: "<%= ENV['RECAPTCHA_SITE_KEY'] %>"
     }
 	</script>
-	<%= IncludeAsset.js '/client/js/nonprofits/cards/edit/page.js' %>
```

Comment:
@camckin10 you had missed this reference. I think we'll remove this file in a separate PR but we still should handle it.